### ISSUE TITLE
Page 15: Adaptor code snippet differs from what is generated

### ISSUE THREAD

**Issue by tylergaw** [abuiles/ember-cli-101-errata#124]
This may have to do with different versions of things. Here's what I have installed:

```
ember-cli: 0.1.5
ember: 1.8.1
```

On page 15:

```
$ ember g adapter application
...
```

The books says that the contents of `app/adapters/application.js` will be:

``` javascript
import DS from 'ember-data';

export default DS.RESTAdapter.extend({
});
```

The contents of `app/adapters/application.js` is:

``` javascript
import ApplicationAdapter from './application';

export default ApplicationAdapter.extend({
});
```

**Comment by tylergaw** [comment]
Seems like this is a change in ember-cli? https://github.com/ember-cli/ember-cli/issues/1764

**Comment by abuiles** [comment]
Thanks for reporting, this looks like a bug in ember-cli, we shouldn't extend if we are creating the file application. Can you please open an issue there? (Also should be an easy fix if you want to contribute to the project :) )

—

**Comment by tylergaw** [comment]
Thanks @abuiles. I'm lost on what the intended behavior is in ember-cli for this. I'm reading through the issues there now to see if something already exists.

**Comment by abuiles** [comment]
If the adapter you are creating is called application, then It should not include "import ApplicationAdapter from './application'" 

—

**Comment by tylergaw** [comment]
A fix for this is out in `0.1.6` https://github.com/ember-cli/ember-cli/releases/tag/v0.1.6

**Comment by abuiles** [comment]
Fixed :)
